L1Trigger/TrackFindingTracklet/plugins/L1FPGATrackProducer.cc

```diff
@@ -398,6 +405,17 @@ void L1FPGATrackProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSe
   const TrackerTopology* const tTopo = &iSetup.getData(esGetTokenTTopo_);
   const TrackerGeometry* const theTrackerGeom = &iSetup.getData(esGetTokenTGeom_);
 
+  // check killing stubs for detector degradation studies
+  // if failType = 0, StubKiller does not kill any modules
+  int failType = 0;
+  if (failScenario_ < 0 || failScenario_ > 9) {
+    std::cout << "Invalid fail scenario! Ignoring input" << std::endl;
```

Comment:
this needs to be changed to use edm::LogVerbatim instead

Reply:
done!